### ISSUE TITLE
Add new --pager-team and --pager-team-suffix flags for PD contact routing

### ISSUE THREAD

**Issue by calebhailey** [sensu/sensu-pagerduty-handler#38]
Inspired by the [sensu-plugins/sensu-plugins-pagerduty contact routing implementation](https://github.com/sensu-plugins/sensu-plugins-pagerduty#usage-of-handler) (see comments regarding `pager_team` check or client attribute). 

### Overview

Users need the ability to override the default Pagerduty API Token on a per-check and/or per-entity basis. This enhancement will enable users to add a single annotation to check or entity resources to configure the behavior of the Pagerduty handler. 

Here's an example check with the `sensu.io/plugins/sensu-pagerduty-handler/config/pager-team` annotation: 

```yaml
---
type: CheckConfig 
api_version: core/v2 
metadata: 
  name: example-check 
  annotations: 
    sensu.io/plugins/sensu-pagerduty-handler/config/pager-team: team_1
``` 

Here's an example `agent.yaml` file with the `sensu.io/plugins/sensu-pagerduty-handler/config/pager-team` annotation set: 

```yaml
---
##
# agent configuration
##
# name: example-agent
# namespace: default
backend-url:
- wss://sensu-backend-0.sensu.sensu-system.svc.cluster.local:8081 
- wss://sensu-backend-1.sensu.sensu-system.svc.cluster.local:8081 
- wss://sensu-backend-2.sensu.sensu-system.svc.cluster.local:8081
subscriptions:
- linux
- docker
- kubernetes
labels:
  org: Sensu, Inc.
annotations: 
  sensu.io/plugins/sensu-pagerduty-handler/config/pager-team: team_1
```

## Proposed changes

- Add a new `--pager-team` flag and corresponding annotation (i.e. `sensu.io/plugins/sensu-pagerduty-handler/config/pager-team`)
- Add a new `--pager-team-suffix` flag (default to `_pagerduty_token`)
- Use the string values of `--pager-team` and `--pager-team-suffix` to read the corresponding Pagerduty API Token value as an environment variable
- Users can set Pagerduty team tokens using the Handler `env_vars` field, or using Sensu's built-in secrets management
- Users can also manage Pagerduty team tokens outside of Sensu (e.g. by editing `/etc/default/sensu-backend` or via config management on the Sensu backend nodes), but we will recommend keeping the config in Sensu

**Comment by calebhailey** [comment]
TBD failure modes: 

- If the team environment variable doesn't exist, do we fallback to the default token or just exit with an error? 
- If the team environment variable exists, but it's not valid, do we retry with the default token or just exit with an error?

**Comment by jspaleta** [comment]
New PR for this feature: #39 
One Note:
using `--pagerduty-team-prefix` instead of suffix  as both `s` and `S` shorthand cmdline arguments are already taken.  Just easier to use prefix with `-P` option

**Comment by portertech** [comment]
Closed by #39
